org.eclipse.buildship.ui/src/main/java/org/eclipse/buildship/ui/view/task/TaskViewState.java

```diff
@@ -54,6 +56,7 @@ public void load() {
         this.linkToSelection = prefs.getBoolean(PREF_LINK_TO_SELECTION, false);
         this.headerNameColumnWidth = prefs.getInt(PREF_HEADER_NAME_COLUMN_WIDTH, 200);
         this.headerDescriptionColumnWidth = prefs.getInt(PREF_HEADER_DESCRIPTION_COLUMN_WIDTH, 400);
+        this.groupProjects = prefs.getBoolean(PREF_GROUP_PROJECTS_BY_HIERARCHY, true);
```

Comment:
Making the hierarchical project representation default breaks the test coverage. I ran the build on our CI, you should be able to check the results [here](https://builds.gradle.org/project.html?projectId=Tooling_Buildship&tab=projectOverview). (credentials: guest/guest). If not, please let me know.

Reply:
I am done.

Reply:
Awesome! I'll soon check the latest changes.